checker/tests/nullness/ConcurrentHashMapTest.java

```diff
@@ -0,0 +1,36 @@
+
+// Test case for issue 1142: https://github.com/typetools/checker-framework/issues/1142
+
+
+import java.util.Iterator;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+class Test{
+
+public static void main(String[] args){
+
+Map<Integer,@Nullable Integer> myMap=new ConcurrentHashMap<>();
+
+myMap.put(1, 1);
+myMap.put(3, 3);
+
+myMap.put(3, null);
+
+}
+}
+
+
+
+/*
+Output without annotated ConcurrentHashMap.java: Nullness Checker passed!
```

Comment:
Properly format your example. Add expected error messages, instead of adding this as a comment.
Make sure the test fails without your change and then passes with your change.